README.md

```diff
@@ -1 +1,76 @@
 # Fluence network environment
+
+[![npm](https://img.shields.io/npm/v/@fluencelabs/fluence-network-environment)](https://www.npmjs.com/package/@fluencelabs/fluence-network-environment)
+
+Officially maintained list of well-known Fluence network nodes. The package is meant to be used in combination with [Fluence JS SDK](https://github.com/fluencelabs/fluence-js).
+
+## Installation
+
+With npm
+
+```bash
+npm install @fluencelabs/fluence-network-environment
+```
+
+With yarn
+
+```bash
+yarn add @fluencelabs/fluence-network-environment
+```
+
+## Usage
+
+Pick a node to connect to the Fluence network.
+
+```typescript
+import { dev } from '@fluencelabs/fluence-network-environment';
+
+export const relayNode = dev[0];
+```
+
+Which can be used to initialize the Fluence client (see [Fluence JS SDK](https://github.com/fluencelabs/fluence-js).)
+
+```typescript
+import { createClient } from '@fluencelabs/fluence';
+
+const client = await createClient(relayNode);
+```
+
+## Known networks
+
+### dev
+
+**Unstable** Several nodes running on the same physical host. Used a lot to test things out. 
+
+### stage
+
+Environment primarily used for staging deployments of various Fluence services.
+
+### FaaS Net
```

Comment:
I think it is better to call it `Test Net`

Reply:
What's the difference between test and stage then?